### PR TITLE
[Intel MKL] Enabling QunatizedConcatV2 with native format

### PR DESCRIPTION
**Pull request by mahmoud-abuzaina**

### CODE REVIEW

**Review by penpornk — COMMENTED** [review]
Thank you for the PR and I'm so sorry for the delay! 
This PR looks good to me. I'll approve it after #45107 is merged.

**Review by penpornk — APPROVED** [review]
Thank you for the PR and sorry for the delay!

**Review by gbaned — APPROVED** [review]

**Pull request merged**
